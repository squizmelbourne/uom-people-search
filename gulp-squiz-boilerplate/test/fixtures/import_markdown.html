<!-- import:markdown test/fixtures/README.md -->