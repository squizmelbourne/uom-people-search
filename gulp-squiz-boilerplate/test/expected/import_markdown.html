<h1 id="heading-1">Heading 1</h1>
<h2 id="heading-2">Heading 2</h2>
<pre><code class="lang-javascript">var code = {
    key: value;
};
</code></pre>
